grasp_apps/draw_x/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>draw_x</name>
+  <version>0.5.0</version>
+  <description>A demo app for draw_x</description>
+  <author email="yu.yan@intel.com">Yu Yan</author>
+  <maintainer email="yu.yan@intel.com">Yu Yan</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <build_depend>rclcpp</build_depend>
+  <build_depend>Eigen3</build_depend>
```

Comment:
can you check if Eigen3 is necessary as build_depend?

Reply:
You are right. Eigen3 is not necessary here.